hasher-matcher-actioner/webapp/src/settings/ThreatExchangeSettingsTab.jsx

```diff
@@ -0,0 +1,244 @@
+              ))}
+        </Row>
+      </Card.Body>
+    </>
+  );
+}
+
+function ThreatExchangePrivacyGroupCard({
+  fetcherActive,
+  inUse,
+  privacyGroupId,
+  privacyGroupName,
+  writeBack,
+  onSave,
+  onDelete,
+  setShowToast,
+  setToastBody,
+}) {
+  const [originalFetcherActive, setOriginalFetcherActive] = useState(
+    fetcherActive,
+  );
+  const [originalWriteBack, setOriginalWriteBack] = useState(writeBack);
+  const [localFetcherActive, setLocalFetcherActive] = useState(fetcherActive);
+  const [localWriteBack, setLocalWriteBack] = useState(writeBack);
+  const onSwitchFetcherActive = () => {
+    setLocalFetcherActive(!localFetcherActive);
+  };
+  const onSwitchWriteBack = () => {
+    setLocalWriteBack(!localWriteBack);
+  };
+
+  return (
+    <>
+      <Col lg={4} sm={6} xs={12} className="mb-4">
+        <Card className="text-center">
+          <Card.Header
+            className={
+              inUse ? 'text-white bg-success' : 'text-white bg-secondary'
+            }>
+            <h4 className="mb-0">{privacyGroupName}</h4>
+          </Card.Header>
+          <Card.Subtitle className="mt-2 mb-2 text-muted">
+            <CopyableTextField text={privacyGroupId} />
+          </Card.Subtitle>
+          <Card.Body className="text-left">
+            <Form>
+              <Form.Switch
+                onChange={onSwitchFetcherActive}
+                id={`fetcherActiveSwitch${privacyGroupId}`}
+                label="Fetcher Active"
+                checked={localFetcherActive}
+                disabled={!inUse}
+              />
+              <Form.Switch
+                onChange={onSwitchWriteBack}
+                id={`writeBackSwitch${privacyGroupId}`}
+                label="Write Back"
+                checked={localWriteBack}
+                disabled={!inUse}
+              />
+            </Form>
+          </Card.Body>
+          <Card.Footer>
+            {localWriteBack === originalWriteBack &&
+            localFetcherActive === originalFetcherActive ? null : (
+              <div>
+                <Button
+                  variant="primary"
+                  onClick={() => {
+                    setOriginalFetcherActive(localFetcherActive);
+                    setOriginalWriteBack(localWriteBack);
+                    setToastBody('Changes are saved!');
+                    setShowToast(true);
+                    onSave({
+                      privacyGroupId,
+                      localFetcherActive,
+                      localWriteBack,
+                    });
+                  }}>
+                  Save
+                </Button>
+              </div>
+            )}
+            {inUse ? null : (
+              <div>
+                <Button
+                  variant="secondary"
+                  onClick={() => {
+                    setToastBody('The privacy group is deleted!');
+                    setShowToast(true);
+                    onDelete(privacyGroupId);
```

Comment:
nit: Same comment as before. Try to centralize behaviors in the handler functions.

I would even suggest allowing the top-level component `ThreatExchangeSettingsTab` to determine whether toasts are to be shown at all or not. 

Low level component like a card should not be able to affect state on their parents. If you find a component altering state of its siblings, reconsider. Since `ThreatExchangeSettingsTab` renders the `Toast` and the `ThreatExchangePrivacyGroupCard`, they are siblings.

Some of this is just experience with react. You may not find this in the standard guides.